mne/evoked.py

```diff
@@ -631,6 +631,34 @@ def plot_joint(self, times="peaks", title='', picks=None,
                                  exclude=exclude, show=show, ts_args=ts_args,
                                  topomap_args=topomap_args)
 
+    def plot_sensors(self, kind='topomap', ch_type=None, show=True):
+        """
+        Plot sensors positions.
+
+        Parameters
+        ----------
+        kind : str
+            Whether to plot the sensors as 3d or as topomap. Available options
+            'topomap', '3d'. Defaults to 'topomap'.
+        ch_type : 'mag' | 'grad' | 'eeg' | 'seeg' | None
+            The channel type to plot. If None, then channels are chosen in the
+            order given above.
+        show : bool
+            Show figure if True. Defaults to True.
+
+        Returns
+        -------
+        fig : instance of matplotlib figure
+            Figure containing the sensor topography.
+
+        Notes
+        -----
+
+        .. versionadded:: 0.12.0
+
+        """
+        return plot_sensors(self.info, kind=kind, ch_type=ch_type, show=show)
```

Comment:
this should be done via a mixin class. It's always the same code for all objects.

now looking at Mixin classes I realize we have a lot and basically all objects derive from pretty much all of them...

I think we should try to have a main one and it would contain this viz function. It would typically fuse ContainsMixin, UpdateChannelsMixin, SetChannelsMixin

thoughts?